tests/tests_service.yml

```diff
@@ -0,0 +1,212 @@
+- name: Test firewalld zones
```

Comment:
Test firewall user defined services

Reply:
I actually typed this one up, so I'm not sure how I managed this one